floweaver/diagram_optimisation.py

```diff
@@ -571,7 +571,7 @@ def optimise_node_positions(sankey_data,
 
     model = straightness_model(sankey_data)
     # FIXME this needs to know what scale we want to use?
-    ys = optimise_position_model(model, scale, wslb=minimum_gap)
+    ys = optimise_position_model(model, scale, wslb=minimum_gap*scale)
```

Comment:
@stratus85 why does the minimum gap need to be scaled? I think it probably makes sense to have this defined in terms of pixels (i.e. the actual gap in the final Sankey diagram)?